src/containers/org/components/OrgActionButton.js

```diff
@@ -94,6 +94,7 @@ const OrgActionButton = ({
 |}) => {
 
   const organizationId :UUID = (organization.id :any);
+  const organizationName :string = (organization.title :any);
```

Comment:
Casting to `any` is not necessary.
```suggestion
  const organizationName :string = organization.title;
```